contrib/cirrus/setup.sh

```diff
@@ -26,6 +26,9 @@ case "$OS_RELEASE_ID" in
         ;;
     ubuntu)
         $SHORT_APTGET update  # Fetch latest package metadata
+        # Upgrading this package doesn't make sense at test-runtime
+        # and has some config. scripts which frequently fail.
+        apt-mark hold grub-efi-amd64-signed
```

Comment:
Can we have something like `$DEBS_HOLD` and use it here?

```sh
[ -z "$DEBS_HOLD" ] || \
￼            apt-mark hold $DEBS_HOLD
```

to be in line with the rest of the code here?

Reply:
Oh, okay, that's a good idea actually...